src/lib/assets/pods.ts

```diff
@@ -349,7 +349,7 @@ export function getModuleSecret(name: string, data: Buffer, hash: string): kind.
   if (secretOverLimit(compressedData)) {
     const error = new Error(`Module secret for ${name} is over the 1MB limit`);
     console.error("Uncaught Exception:", error);
```

Comment:
```suggestion
    const error = new Error(`Module secret for ${name} is over the 1MB limit`);
```

In `watch-processor.ts` we started using `Error`'s `cause` attribute to provide a slightly more useful message to users and processors of the error message as we move up the call-stack.

I think we can drop the `Uncaught Exception` console message based upon the sample output in `npx pepr deploy`.